orchestra/todos/serializers.py

```diff
@@ -85,14 +85,17 @@ class BulkTodoSerializer(serializers.ModelSerializer):
     }
 
     step = StepField()
+    additional_data = serializers.JSONField(required=False)
 
     # TODO(murat): Remove this validation when project
     # becomes a required field in models
     def validate(self, data):
-        if data.get('project') is None:
-            raise serializers.ValidationError(
-                {'project': ['project should be supplied.']}
-            )
+        step = data.get('step')
```

Comment:
What if both project and step are None? Is there a case when this will happen?

Reply:
When we are updating a todo partially, the update payload may be lacking both, `step` and `project`, and can only contain, say, `id` and `order`.